tracing/tracing.go

```diff
@@ -36,7 +36,7 @@ func NewFromEnv(serviceName string) io.Closer {
 
 	if cfg.Sampler.SamplingServerURL == "" && cfg.Reporter.LocalAgentHostPort == "" {
 		fmt.Printf("Jaeger tracer disabled: No trace report agent or config server specified\n")
-		return ioutil.NopCloser(nil)
+		return ioutil.NopCloser(nil), nil
```

Comment:
Feels like this should be an error return for consistency, perhaps to an exported variable so the caller can check which kind of error.

Reply:
Will change